components/upload/UploadList.tsx

```diff
@@ -226,28 +226,27 @@ export default class UploadList extends React.Component<UploadListProps, any> {
       } else {
         message = (file.error && file.error.statusText) || locale.uploadError;
       }
-      const iconAndPreview =
-        file.status === 'error' ? (
-          <Tooltip title={message}>
-            {icon}
-            {preview}
-          </Tooltip>
-        ) : (
-          <span>
-            {icon}
-            {preview}
-          </span>
-        );
-
-      return (
-        <div className={infoUploadingClass} key={file.uid}>
+      const iconAndPreview = (
+        <span>
+          {icon}
+          {preview}
+        </span>
+      );
+      const dom = (
+        <div>
           <div className={`${prefixCls}-list-item-info`}>{iconAndPreview}</div>
           {actions}
           <Animate transitionName="fade" component="">
             {progress}
           </Animate>
         </div>
       );
+
+      return (
+        <div key={file.uid} className={infoUploadingClass}>
+          {file.status === 'error' ? <Tooltip title={message}>{dom}</Tooltip> : <span>{dom}</span>}
```

Comment:
`Tooltip` should have some space

![image](https://user-images.githubusercontent.com/14831261/68655802-a5d41280-056b-11ea-9618-a4f8e7e72642.png)

Reply:
ok